routes/userRouter.js

```diff
@@ -1,10 +1,16 @@
 const express = require("express");
 const userController = require("../controllers/userController");
+const loginRequired = require("../utils/auth");
 
 const router = express.Router();
 
 router.post("/signup", userController.signUp);
 router.post("/login", userController.signIn);
 router.post("/emailcheck", userController.emailCheck);
+router.get("/mypage", loginRequired, userController.myPageShow);
+router.get("/addressshow", loginRequired, userController.myAddress);
```

Comment:
```suggestion
router.get("/address", loginRequired, userController.myAddress);
```
이미 GET + /address 의 조합으로 주소를 불러온다고 직관적으로 보이기 때문에 동사는 작성될 필요가 없습니다. 위와 같이 수정되면 조금 더 RESTful한 endpoint가 될 것 같아요!

Reply:
네 수정했습니다 !